src/__tests__/lib/containers/entity/metadata/EntityModal.test.tsx

```diff
@@ -108,6 +108,28 @@ describe.skip('EntityModal tests', () => {
     ).not.toBeInTheDocument()
   })
 
+  it('Does not show the edit button for annotations when looking at an older version', async () => {
```

Comment:
All of these tests are skipped because they are unstable, but this was working locally.